### ISSUE TITLE
Add support for exporting CKEditor fields (with config)

### ISSUE THREAD

**Issue by rungta** [engram-design/FieldManager#100]
### What are you trying to do?

Exporting CKEditor fields.

### What's your proposed solution?

Craft has started storing CKEditor field configs separately in the project config. These are currently not included in the field export, only a reference to the config is included. Eg:

```json
    {
        "name": "Rich Text",
        "handle": "richtext",
        "instructions": null,
        "required": null,
        "searchable": true,
        "translationMethod": "site",
        "translationKeyFormat": null,
        "type": "craft\\ckeditor\\Field",
        "settings": {
            "ckeConfig": "b317fb51-9087-4bb2-b4ce-7dfc46e566c1",
            "wordLimit": null,
            "showWordCount": false,
            "availableVolumes": "*",
            "availableTransforms": "",
            "defaultTransform": null,
            "enableSourceEditingForNonAdmins": false,
            "showUnpermittedVolumes": false,
            "showUnpermittedFiles": false,
            "purifierConfig": null,
            "purifyHtml": true,
            "columnType": "text"
        }
    }
```

Would be good to have the full config included in the export so that it can be imported on a different Craft project with the config intact.

**Comment by engram-design** [comment]
Added for the next release. To get this early, run `composer require verbb/field-manager:"dev-craft-4 as 3.0.8"`.

**Comment by rungta** [comment]
Thanks Josh!

**Comment by engram-design** [comment]
Fixed in [3.0.9](https://github.com/verbb/field-manager/releases/tag/3.0.9)
